pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -79,8 +82,13 @@ func newTotalAccountWatcher(
 
 // TotalAccountWatcher will trigger AwsLimitUpdate every `scanInternal` and only stop if the operator is killed or a
 // message is sent on the stopCh
-func (s *totalAccountWatcher) Start(log logr.Logger, stopCh <-chan struct{}) {
+func (s *totalAccountWatcher) Start(log logr.Logger, stopCh <-chan struct{}, client client.Client, watchInterval time.Duration) {
 	log.Info("Starting the totalAccountWatcher")
+	// This should run only the first time
+	if !TotalAccountWatcherInitialized {
+		Initialize(client, watchInterval)
```

Comment:
Big +1 combining `Initialize` and `Start`. You could take it a step further and make `Initialize` private.

Reply:
yup, good idea ;)